mlocal/frags/Makefile.stub

```diff
@@ -69,7 +69,7 @@ cniplugins: $(BUILDDIR)/vendors-done
 	@for p in `go list -f '{{if eq .Name "main"}}{{.ImportPath}}{{end}}' ../vendor/$(cni_plugins_REPO)/...|grep -v sample`; do \
 		if [ ! -f $(cni_builddir)/`basename $$p` ]; then \
 			echo " CNI PLUGIN" `basename $$p`; \
-			go build -o $(cni_builddir)/`basename $$p` $$p; \
+			go build -ldflags="-s" -o $(cni_builddir)/`basename $$p` $$p; \
```

Comment:
I had looked into that, but couldn't find what -w does in the ld man page.  Do you know?  -s is for stripping symbols, which is what is needed because otherwise rpm tried to extract the symbols and failed because it couldn't identify the "build id".

Reply:
IIRC, `-w` removes the DWARF debug info.

Reply:
Ok I added -w